DesktopModules/Vanjaro/UXManager/Extensions/Menu/Pages/Views/Setting/Detail.html

```diff
@@ -156,7 +156,7 @@ <h6 class="addpage-icon-text mt-md mb-xs">[L:PageTypeURL]</h6>
             <uielement name="DisplayInMenu" container-attr-ng-if="ui.data.IsDefaultLocale.Options" container-attr-ng-show="ShowPage_detailsTab && Show_Tab" ngmodel="ui.data.PagesTemplate.Options.includeInMenu" attr-class="toggle_right" control="toggle"></uielement>
             <uielement name="MakePublic" container-attr-ng-if="ui.data.IsDefaultLocale.Options && ShowPage_detailsTab && Show_Tab && pid==0" attr-class="toggle_right" control="toggle"></uielement>
             <uitemplate>
-                <div class="form-group" mn-ng-if="ui.data.IsDefaultLocale.Options" mn-ng-show="ShowPage_detailsTab&&Show_Tab &&(ui.data.PagesTemplate.Options.pageType=='Standard'||ui.data.PagesTemplate.Options.pageType=='Folder')">
+                <div class="form-group" mn-ng-if="ui.data.IsDefaultLocale.Options" mn-ng-show="ShowPage_detailsTab&&Show_Tab">
```

Comment:
Why are we adding additional conditions here? Why not update ShowPage_detailsTab model?

Reply:
Show_Tab by default false, at that time we will show Choose Template UI and hide all tab of the page settings
and ShowPage_detailsTab using to hide and show individual tab